### ISSUE TITLE
text2pointcloud with larger models

### ISSUE THREAD

**Issue by AmanKishore** [openai/point-e#13]
Hello, when I try to run text2pointcloud with 300M or 1B models I'm getting the following error: 
```\KeyError                                  Traceback (most recent call last)
Cell In[8], line 6
      4 # Produce a sample from the model.
      5 samples = None
----> 6 for x in tqdm(sampler.sample_batch_progressive(batch_size=1, model_kwargs=dict(texts=[prompt]))):
      7     samples = x

File ~/.local/lib/python3.8/site-packages/tqdm/notebook.py:259, in tqdm_notebook.__iter__(self)
    257 try:
    258     it = super(tqdm_notebook, self).__iter__()
--> 259     for obj in it:
    260         # return super(tqdm...) will not catch exception
    261         yield obj
    262 # NB: except ... [ as ...] breaks IPython async KeyboardInterrupt

File ~/.local/lib/python3.8/site-packages/tqdm/std.py:1195, in tqdm.__iter__(self)
   1192 time = self._time
   1194 try:
-> 1195     for obj in iterable:
   1196         yield obj
   1197         # Update and possibly print the progressbar.
   1198         # Note: does not call self.update(1) for speed optimisation.

File ~/mirage-dev/aman/deep-learning/point-e/point_e/diffusion/sampler.py:130, in PointCloudSampler.sample_batch_progressive(self, batch_size, model_kwargs)
...
    318 _ = batch_size
    319 with torch.no_grad():
--> 320     return dict(embeddings=self.clip.embed_images_grid(model_kwargs["images"]))

KeyError: 'images'
```

Works fine with image2pointcloud

Appreciate the help!

**Comment by unixpickle** [comment]
The main method explored in the paper is a two stage pipeline: text->image followed by image->3D. We do not release the text->image model, but do show that the image->3D model can work with images from other text->image models sometimes (shown in the appendix).

The direct text->pointcloud model is from an ablation we ran, which showed that the text->image step greatly increased performance compared to a direct text->pointcloud model. We only trained a small text->pointcloud model for this ablation and did not scale it up.

![Screenshot_20221221-105204](https://user-images.githubusercontent.com/406021/208974144-dd7f6289-0e83-49a0-8df1-41d57e8c47a3.png)
